IoTuring/Entity/Deployments/Fanspeed/Fanspeed.py

```diff
@@ -0,0 +1,261 @@
+import psutil
+
+from IoTuring.Entity.Entity import Entity
+from IoTuring.Entity.EntityData import EntitySensor
+from IoTuring.Entity.ValueFormat import ValueFormatterOptions
+from IoTuring.MyApp.SystemConsts import OperatingSystemDetection as OsD
+from IoTuring.Configurator.MenuPreset import MenuPreset
+
+
+supports_win_fanspeed = False
+supports_linux_fanspeed = True
+supports_macos_fanspeed = False
+
+KEY_FANSPEED = "fanspeed"
+KEY_FANLABEL = "fanlabel"
+
+FALLBACK_PACKAGE_LABEL = "controller"
+FALLBACK_SENSOR_LABEL = "fan"
+
+CONFIG_KEY_CONTROLLER = "controller"
+CONFIG_KEY_THRESHOLD = "threshold"
+
+FANSPEED_DECIMALS = 0
+
+
+class Fanspeed(Entity):
+    """Entity to read fanspeed"""
+
+    NAME = "Fanspeed"
+
+    def Initialize(self) -> None:
```

Comment:
You should `raise` an exception for unsupported OS in `Initialize`, so the entity would be unloaded, like here: https://github.com/richibrics/IoTuring/blob/main/IoTuring/Entity/Deployments/Temperature/Temperature.py#L52

Also if the system doesn't have a fan, e.g. a RaspberryPi, an exception should be raised similarly